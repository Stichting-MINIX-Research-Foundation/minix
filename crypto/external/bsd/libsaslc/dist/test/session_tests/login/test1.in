LOGIN
2
AUTHID test
PASSWD test2


