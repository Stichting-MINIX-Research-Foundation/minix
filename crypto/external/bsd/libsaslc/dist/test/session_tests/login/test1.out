mechanism: session:
test
test2
